nixos/modules/services/hardware/interception-tools.nix

```diff
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
```

Comment:
You also need to add this module to `nixos/modules/module-list.nix` as well.

Reply:
Thank you, fixed. I missed this because although I tested the packages themselves, I didn't test the service and just copied the file over, sorry. :expressionless:

Reply:
As in, I did test the service, just not from under nixpkgs, but from importing it in `configuration.nix`.